public/app/features/alerting/unified/components/rule-editor/rule-types/RuleTypePicker.tsx

```diff
@@ -6,16 +9,15 @@ import { RuleFormType } from '../../../types/rule-form';
 import { GrafanaManagedRuleType } from './GrafanaManagedAlert';
 import { MimirFlavoredType } from './MimirOrLokiAlert';
 import { RecordingRuleType } from './MimirOrLokiRecordingRule';
-import { css } from '@emotion/css';
-import { GrafanaTheme2 } from '@grafana/data/src';
-import { Stack } from '@grafana/experimental';
 
 interface RuleTypePickerProps {
   onChange: (value: RuleFormType) => void;
   selected: RuleFormType;
+  enableGrafana: boolean;
```

Comment:
**suggestion:** I think a better pattern would be to pass an array with enabled `RuleFormType`

`enabled: RuleFormType[]`

We can then check for `enabled.includes(RuleFormType.grafana)` etc.

What do you think?

Reply:
Good idea! I'll update